### ISSUE TITLE
Prevent Autocommand Pileup

### ISSUE THREAD

**Issue by angelic-sedition** [5digits/dactyl#29]
Putting an `autocmd DownloadPost` in the rc will result in that autocommand being executed extra times for every window that is opened. Is there some way to have something in the rc executed only once (I have tried `au Enter...` without success)? If not, it would be nice if either this was added or something along the lines of vim's augroups/autocmd! for clearing.

**Comment by willsALMANJ** [comment]
For the most part, Pentadactyl configuration is done on a per-window basis by sourcing `.pentadactylrc` each time a new window is launched. For web-page specific autocommands, the current behavior makes sense since you would would a `PageLoad` autocommand defined within each window. However, for autocommand conditions based on browser-wide events like `DownloadPost`, you would only want one autocommand per session. I think this requires a patch to somehow define session-wide autocommands (and I think whoever created the `DownloadPost` and other browser-wide autocommand event triggers must always work within a single window).

I'm not offering a solution here -- just framing the problem because it took me a little while to see what the issue was with the way things currently work.

**Comment by angelic-sedition** [comment]
Yes I agree. It's worth looking at [this](https://groups.google.com/forum/#!topic/pentadactyl/rwrGG4njsq8) on the pentadactyl google group for a workaround.

I've also noticed that after opening a new window, my `DownloadPost` autocmd will show up twice with `:js group.autocmd.get()`, but `Enter` autocmds I have still only show up once.
